lookup_plugins/sequence.py

```diff
@@ -1,215 +0,0 @@
-# (c) 2013, Jayson Vantuyl <jayson@aggressive.ly>
```

Comment:
We'll need to update docs to say we require 1.9.4 at minimum, but +1 to removing this hack.